en/docs/learn/integration-with-bank.md

```diff
@@ -0,0 +1,450 @@
+        2. Replace the `https://localhost:9443/api/openbanking/berlin/backend/services/accounts` URL.
+    - For CoF submission request:
+        1. Open the `<APIM_TOOLKIT_HOME>/repository/resources/apis/berlin.group.org/ConfirmationOfFunds/cof-consents-dynamic-endpoint-sequence-1.3.6` file.
+        2. Replace the `https://localhost:9443/api/openbanking/berlin/backend/services/funds-confirmations` URL.
+
+
+## Accounts Retrieval Endpoints
+
+In some banks, some PSUs may have a certain number of accounts, but not all accounts have the ability to be shared
+externally or to make a payment online. Therefore, in a bank, the shareable account list and the payable account list can
+either be the same or different.
+
+In the default WSO2 Open Banking Berlin Toolkit, at least two APIs are expected to return shareable and payable accounts.
+When passing the `user_id` a JSON response must be returned. Then it automatically loads the accounts list on the consent page.
+
+### Payable Accounts API
+
+The back-end endpoint for the Payable Accounts API is used to retrieve the payable accounts of the user during the
+authentication flow. This API validates the provided account against actual payable accounts. When invoking this
+API, the `consentId` and `userId` (PSU ID) must be sent in the URL as query parameters.
+
+The back end endpoints for payable and sharable accounts retrieval can be configured in the
+`<IS_HOME>/repository/conf/deployment.toml` file as follows.
+By default, the mock back end that is deployed in the API Manager is configured in this file.
+
+``` toml
+[open_banking_berlin.consent]
+payable_account_retrieval_endpoint = "https://<APIM_HOST>:9443/api/openbanking/berlin/backend/services/v130/accounts/payable”
+```
+
+Required parameters can be passed as query parameters to those endpoints. Given below is an example of configuring the
+endpoint to retrieve payable accounts:
+
+``` toml
+[open_banking_berlin.consent]
+payable_account_retrieval_endpoint = "https://<APIM_HOST>:9443/api/openbanking/berlin/backend/services/v130/accounts/payable/{user}”
+```
+
+The response from the API should be formatted as follows:
+
+``` json
+{
+   "customerIdentification":{
+      "customerIdentification":"JEAN123",
+      "userIdentification":"user"
+   },
+   "accounts":[
+      {
+         "iban":"DE12345678901234567890",
+         "isDefault":true,
+         "currency":"USD"
+      },
+      {
+         "bban":"BARC12345612345678",
+         "isDefault":true,
+         "currency":"USD"
+      },
+      {
+         "pan":"5409050000000000",
+         "isDefault":true,
+         "currency":"USD"
+      },
+      {
+         "maskedPan":"123456xxxxxx1234",
+         "isDefault":true,
+         "currency":"USD"
+      }
+   ]
+}
+```
+
+### Shareable Accounts API
+
+The Shareable Accounts API is used for the following two purposes:
+
+1. To validate the debtor account if the PISP sends it in the initiation request
```

Comment:
- We use the sharable accounts endpoint in the AISP flow to retrieve the account IDs from the bank during the authorization flow to display the account IDs on the consent page. In the bank offerred consent scenario, it will retrieve all available accounts for the user and display on the consent page so the user can select accounts. In a scenario of account IDs are already provided in the consent, the endpoint will be used to check whether the accounts in the consent are available in the bank backend.
 - We use payable accounts endpoint to validate whether the debtor account provided in the PIS (payment)/PIIS (confirmation of funds) consent are available in the bank backend.

Reply:
Updated according to this in 2758203e7a0e82c38aa799092d389531d787e250